Duvet/SheetStatusBarStyleProvider.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+
+/// Controls the source of truth for a `SheetViewController`'s status bar style.
+public enum SheetStatusBarStyleProvider {
+    /// Use the `SheetViewController`'s preferred status bar style.
+    case sheetViewController
```

Comment:
In what case(s) would you pick this over `sheetItems`? Or is this mostly for backwards compatibility?

Reply:
Yup! The only purpose this serves is to use it as the default to preserve the existing behavior

Reply:
This feels to me like a bug that we should fix across the board rather than a new feature that you have to opt into.

Unless you've customized the `preferredStatusBarStyle` of your view controller that you're showing in the sheet, it should return `.default` by default, right? So that should match the behavior of this `sheetViewController` option. So the only case we would break would be the case where the view controller had already overridden `preferredStatusBarStyle`, in which case I think we should honor that style automatically. What do you think about this?

Reply:
I'm cool with that. I was not sure if the existing functionality was intentional, so I didn't want to change the default behavior. In that case, I'll adapt this so that it's just a bug fix. 

I have one more bug fix and then we can cut a release bumping the patch version number

Reply:
Sounds good, thanks!